examples/linux64/hello-make/hello.c

```diff
@@ -0,0 +1,51 @@
+// Currently, it's not possible to specify executable entrypoint to ppci
+// linker - so far, the entrypoint is at the beginning of the code segment.
+// And functions appear in the code segment in the order of appearance in
+// the source file, even if "apperance" is merely a prototype. Thus, the
+// first function should be the "startup" function, and it should use
+// inline asm to call other functions (because currently with PPCI, it's
+// not possible to call a function without prototype, and putting a
+// prototype before the startup function will push it down the code segment,
+// precluding from running as entrypoint).
+int _startup()
+{
+    asm("call main");
```

Comment:
Note that I recently added multiline asm syntax support if you want to use it:
```c
asm(
  "call main\n"
  "jmp exit\n"
);
```

Reply:
> Note that I recently added multiline asm syntax

Well, I, multiline inline asm syntax wit those explicit \n's isn't particularly pretty, so given a choice, I used per-line asm's. But given that register info applies to the entire asm block, there's no choice, so will reformat as you suggest.

Reply:
Oh, I mean, here there's no unavoidable need to reformat, but for consistency...

Reply:
> Well, I, multiline inline asm syntax wit those explicit \n's isn't particularly pretty

Fully agree, this is ugly, but it is an artifact of string literals being glued together at the lexer level in C, so that's probably why it is required.